portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java

```diff
@@ -2596,6 +2599,28 @@ userId, _EMAIL_TYPE_EXPIRED, _buildEntryURL(latestFileVersion),
 		}
 	}
 
+	private void _expireFileVersion(
+			DLFileEntry fileEntry, DLFileVersion fileVersion, long userId,
+			Map<String, Serializable> workflowContext,
+			ServiceContext serviceContext)
```

Comment:
I think parameter `serviceContext` should go before `workflowContext` (sorting reasons) and `userId` at the start

Reply:
I use the same sort as used in existing calling method:

```
private void _expireFileEntriesByCompanyId(
			long companyId, Date expirationDate,
			Map<String, Serializable> workflowContext,
			ServiceContext serviceContext)
		throws PortalException {
```

Reply:
Why is a breaking change??

Default behavior is the same.

Reply:
I could move the userId if you want, anyway

Reply:
OK, keep using the same order then. But at least I would move `userId` parameter.

Reply:
updated here: https://github.com/liferay-lima/liferay-portal/pull/4839/commits/a074ae1470b618ab8d86d31c6a6a05216a31cc32